test/common/testvm.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.arch = "x86_64"
 
         self.image = image or testinfra.DEFAULT_IMAGE
+        self.atomic_image = self.image in ["rhel-atomic", "fedora-atomic", "continuous-atomic"]
```

Comment:
This list was repeated in the existing code several times, so this centralizes that list. Other places used `"atomic" in m.image` which might lead to unintended test skips, and the tests should be consistent in how they check for "am I running on a known atomic image".